lib/digital_public_works/projects.ex

```diff
@@ -18,65 +18,68 @@ defmodule DigitalPublicWorks.Projects do
       [%Project{}, ...]
 
   """
-  def list_projects(_user \\ nil, _search \\ nil)
+  def list_projects(), do: all() |> fetch()
 
-  def list_projects(nil, search) do
-    Project
-    |> where(is_public: true)
-    |> filter_projects(search)
-    |> Repo.all()
-  end
+  def list_published_projects(search \\ nil), do: all() |> published() |> fetch(search)
 
-  def list_projects(%User{is_admin: true}, search) do
-    Project
-    |> filter_projects(search)
-    |> Repo.all()
-  end
+  def list_featured_projects(), do: all() |> featured() |> fetch()
```

Comment:
Loving how concise and expressive this got. Main interface is to use the predefined methods to get the specific group of projects you want, and those are easy to add to by composing the queries together. Very nice.